packages/templates/src/raw/12-hours-in-barcelona/template.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { TemplateData } from '../../types';
```

Comment:
@barklund @swissspidy I've added file declaration for the 5 specific templates that fail with the TypeScript inferred type being too narrow for `Groups` for some reason. Still didn't figure out why it doesn't work as for the other templates, don't want to spend too much time on it either unless it becomes a priority. See [c1892fd](https://github.com/GoogleForCreators/web-stories-wp/pull/12429/commits/c1892fd965f43a9d16f466433ea22b3e08dab307)

The rest of the templates work ok as they are.